src/helpers/helpers.dom.ts

```diff
@@ -94,10 +105,14 @@ function getCanvasPosition(e, canvas) {
 /**
  * Gets an event's x, y coordinates, relative to the chart area
  * @param {Event|ChartEvent} evt
- * @param {dom.Chart} chart
+ * @param {Chart} chart
  * @returns {{x: number, y: number}}
```

Comment:
now we should remove all type annotations from jsdoc (so it becomes tsdoc)

Reply:
removed all type annotations, is this correct? I guess the types are already explicit on the function definition